common/components/framework-response/template.njk

```diff
@@ -1,6 +1,8 @@
 {% if params.value and params.valueType %}
   {% if params.valueType === 'object' %}
-    {{ params.value.option }}{{ " — " + params.value.details if params.value.details }}
+    <p class="govuk-!-font-size-16">
```

Comment:
I take it that the array might be output somewhat differently. Otherwise, could the p element not wrap around everything inside the `{% if params.value and params.valueType %}` conditional?

Reply:
Indeed. `collections` and `arrays` are displayed using an unordered list element and it didn't make sense semantically to nest those `ul`s inside a `p` element to.

Reply:
Bit hard to spot actually because this diff folds away the existing `ul` elements. Line 7 of this file is an example.